libtinyaes/Makefile

```diff
@@ -0,0 +1,20 @@
+#
+# Makefile for Tiny AES library
+#
+# Copyright 2024 Phoenix Systems
+# Author: Aleksander Kaminski
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+#
+
+NAME := libtinyaes
+LOCAL_PATH := $(call my-dir)
+
+# TODO: STM32L4 HW implementation disabled for now, make it conditional somehow.
+# Even on STM32L4 target we might want a pure SW implementation.
+SRC := $(filter-out aes_hw_stm32l4.c, $(wildcard $(LOCAL_PATH)*.c))
```

Comment:
Shouldn't it be `SRCS` or `LOCAL_SRCS` respectively ? 
btw I would also add `libtinyaes` to [DEFAULT_COMPONENTS](https://github.com/phoenix-rtos/phoenix-rtos-corelibs/blob/master/Makefile#L14)

Reply:
Thx, done